src/meson.build

```diff
@@ -76,6 +73,25 @@ if appstream.version().version_compare('>=0.16')
     args += '--define=HAS_APPSTREAM_0_16'
 endif
 
+if get_option('packagekit_backend')
+    appcenter_files += files(
+        'Core/PackageKitBackend.vala',
+        'Core/Task.vala',
+    )
+
+    args += '--define=PACKAGEKIT_BACKEND'
+endif
+
+if get_option('ubuntu_drivers_backend')
+    if not get_option('packagekit_backend')
+        error('Option ubuntu_drivers requires option packagekit_backend')
+    endif
+
+    appcenter_files += files('Core/UbuntuDriversBackend.vala')
+
+    args += '--define=UBUNTU_DRIVERS_BACKEND'
```

Comment:
I don't see this getting used in this PR. What is the intent with this define?

Reply:
This is used in `src/Core/BackendAggregator.vala` to check if the backend should be added.